product_attribute_multi_type/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product attribute multi type
+============================
+
+This module extends product attributes, adds 2 new types: range & numeric, and
+also the option of defining if it is required and a default value for the attribute.
+It brings the basic functionnality that can be used then by more fonctional modules.
```

Comment:
Fix directly in https://github.com/OCA/product-attribute/commit/9c855dee31524535946b31ea21fa47022280e50a